llvm/test/Analysis/BasicAA/size-overflow.ll

```diff
@@ -0,0 +1,14 @@
+; RUN: opt -passes=aa-eval -print-all-alias-modref-info -disable-output 2>&1 | FileCheck %s
+
+target datalayout = "p:32:32"
+
+; Make sure that using a LocationSize larget than the index space does not
+; assert.
+
+; Just Mod:  Ptr: i32* %gep	<->  call void @llvm.memset.p0.i64(ptr %p, i8 0, i64 68719476736, i1 false)
```

Comment:
Missing `CHECK` below?
```suggestion
; CHECK: Just Mod:  Ptr: i32* %gep	<->  call void @llvm.memset.p0.i64(ptr %p, i8 0, i64 68719476736, i1 false)
```

Reply:
Oops, looks like I never actually ran the new test...